client/src/components/comparison/BlankUserOutfitCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React, {useEffect, useState} from 'react';
+
+const BlankUserOutfitCard = (props) => {
+
+  const defaultPic = 'https://i.imgur.com/R7mqXKL.png';
```

Comment:
Compress this image and serve locally instead of calling an external site to retrieve